src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -36,6 +36,12 @@ set(NATIVECOMPRESSION_SOURCES
     pal_zlib.c
 )
 
+if (HOST_WIN32 OR CLR_CMAKE_TARGET_WIN32)
```

Comment:
Can this be just `CLR_CMAKE_TARGET_WIN32`?

Reply:
I'm not sure - `HOST_WIN32` was existing condition.  I'll have a look.